app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
   <meta name="msvalidate.01" content="3667D2FEB238C1BF174F3CB1AE0D3C16" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true %>
-  <title><%= content_for?(:title) ? content_for(:title) : "Libraries.io - The Open Source Discovery Service" %></title>
+  <title><%= content_for?(:title) ? content_for(:title) + " - " : ""%>Libraries.io - security & maintenance data for open source software</title>
```

Comment:
this change is the secret sauce

the default blurb was moved outside the else case of the ternary, so it's always present